src/main/webapp/WEB-INF/themes/desktop/xform/article-transform.xsl

```diff
@@ -1617,40 +1616,40 @@
     </xsl:element>
   </xsl:template>
 
-  <!-- 1/4/12: nlm contains alt-text (suppressed here, processed within graphic|inline-graphic) -->
+  <!-- nlm contains alt-text (suppressed here, processed within graphic|inline-graphic) -->
 
-  <!-- 1/4/12: Ambra modifications -->
-    <xsl:template match="list">
+  <!-- Ambra modifications -->
+  <xsl:template match="list">
+    <xsl:call-template name="newline1"/>
+    <xsl:if test="title">
+      <h5><xsl:value-of select="title"/></h5>
+    </xsl:if>
+    <xsl:choose>
+      <xsl:when test="@list-type='bullet'">
         <xsl:call-template name="newline1"/>
```

Comment:
This whole block should be indent changes but it resulted in one line post-edit having the same indent as another line pre-edit which makes the diff look awkward.